store/src/main/java/org/apache/rocketmq/store/logfile/DefaultMappedFile.java

```diff
@@ -560,7 +560,7 @@ public void warmMappedFile(FlushDiskType type, int pages) {
         ByteBuffer byteBuffer = this.mappedByteBuffer.slice();
         long flush = 0;
         // long time = System.currentTimeMillis();
-        for (long i = 0, j = 0; i < this.fileSize; i += DefaultMappedFile.OS_PAGE_SIZE, j++) {
+        for (long i = 0; i < this.fileSize; i += DefaultMappedFile.OS_PAGE_SIZE) {
```

Comment:
j is used in the code commented below

Reply:
now the code commented below doesn't work

Reply:
> j is used in the code commented below

so，It's not necessary to delete j